packages/odyssey-react-mui/src/labs/SideNav/NavAccordion.tsx

```diff
@@ -92,8 +88,9 @@ const AccordionSummaryContainer = styled(MuiAccordionSummary, {
   isDisabled?: boolean;
 }>(({ odysseyDesignTokens, isCompact, isDisabled }) => ({
   borderRadius: odysseyDesignTokens.BorderRadiusMain,
-  paddingBlock: odysseyDesignTokens.Spacing3,
+  paddingBlock: odysseyDesignTokens.Spacing2,
   paddingInline: odysseyDesignTokens.Spacing4,
+  minHeight: odysseyDesignTokens.Spacing5,
```

Comment:
We can remove this min-height

Reply:
@bryancunningham-okta the height of items, and inside the NavAccordion seem to vary with this style. Do you recommend a better solution?
<img width="1315" alt="image" src="https://github.com/user-attachments/assets/30ad63eb-25e7-465c-ad9d-6fa279cba437">

Reply:
What about `height: 100%; max-height: pixels`?

Reply:
https://github.com/okta/odyssey/pull/2405/files#r1831682389

Reply:
We don't need either. The content of the item sets the height correctly

Reply:
I see what you are saying now. I'd say just do `minHeight: "unset"` then